### ISSUE TITLE
sequence numbering on `nextstrain` trees

### ISSUE THREAD

**Issue by jbloom** [dms-vep/RABV_Pasteur_G_DMS#47]
@Caleb-Carr @arjunaditham, when the `nextstrain` trees at [https://nextstrain.org/groups/jbloomlab/dms/rabies-G?c=gt-G_346](https://nextstrain.org/groups/jbloomlab/dms/rabies-G?c=gt-G_346) are colored by amino-acid identity at specific sites, this does not match up to what is shown in the paper. Maybe this due to incongruent numbering schemes, but this needs to be fixed. If the main point of the tree is to show the DMS data then the sites need to be numbered in the same way as described in the paper.

Can you look into what is going on here and fix it?

**Comment by arjunaditham** [comment]
we'll look into fixing this. What has happened is that the tree is numbered based on the site in the overall rabies G sequence, and we want the numbering convention to match the ectodomain convention only. For example, when sites N336 and R346 are wanted, we need to query sites 355 and 365 respectively. 

Here's for R346: https://nextstrain.org/groups/jbloomlab/dms/rabies-G?c=gt-G_365

**Comment by jbloom** [comment]
Yes, we should try to harmonize the numbering. Once this is done, @Caleb-Carr, the description should be updated to indicate the numbering scheme used.

**Comment by Caleb-Carr** [comment]
@jbloom @arjunaditham Just to confirm, tree numbering should be updated to match the DMS numbering (i.e., the first position will now be -2) and all sites that were not included in DMS should be removed from the `nextstrain` tree?

**Comment by jbloom** [comment]
I'm not sure how you do this via NextStrain, but ideally (if possible?) you would not remove any sites but rather just use the same numbering system in the DMS. Eg, sites -19, -18, ... would still be there, but entering a site number would be correct.

Note also that for this to work you need to ensure the alignment has no gaps relative to the Pasteur G. Arjun thought that was the case, but check.

**Comment by Caleb-Carr** [comment]
I'll look into specifying site numbering because I'm also not sure if this can be done.

And yes, all gaps are removed relative to the Pasteur G.

**Comment by jbloom** [comment]
Probably @huddlej would know if this is possible in `nextstrain` as it also comes up in numbering influenza HA, where the numbering scheme is not sequential w respect to the 1, 2, numbering of the protein.

**Comment by jbloom** [comment]
@Caleb-Carr, you do this by specifying the appropriate annotations in a Genbank a GTF / GFF file, as [here](https://github.com/nextstrain/seasonal-flu/blob/master/config/h3n2/ha/genemap.gff) for the `nextstrain` H3N2 build. 

You then pass this to the `augur translate` command in your pipeline.

For your case, you can just annotate `G ectodomain` as a feature in the Genbank or GTF/GFF file. If you want, you can also annotate `full-length G` as a separate larger feature. Then just add text in the description of the tree explaining the numbering schemes. Just like for the `nextstrain` H3N2 build both annotations should show up as things you can color by.
